.github/workflows/appsignals-e2e-test.yml

```diff
@@ -156,55 +165,63 @@ jobs:
             sleep 10
           done
 
+      # This steps increases the speed of the validation by creating the telemetry data in advance
+      - name: Call all test APIs
+        continue-on-error: true
+        run: |
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/outgoing-http-call/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/aws-sdk-call/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/client-call/
+
       # Validation for app signals telemetry data
       - name: Call endpoint and validate generated EMF logs
         id: log-validation
-        working-directory: test/validator
-        run: ./gradlew run --args='-c log-validation.yml
+        if: steps.endpoint-check.outcome == 'success' && !cancelled()
+        run: ./gradlew testing:validator:run --args='-c eks/log-validation.yml
```

Comment:
So was this just failing before because of a) incorrect command specification and b) incorrect path? Can you provide a failed test run for comparison?

Also is the command missing a `'`? e.g. `--args='-c eks/log-validation.yml` -> `--args='-c eks/log-validation.yml'`.

Reply:
No, the test was previously silently failing because our solution to replace the CW Agent operator was deprecated due to use of the EKS addon instead of deployment configurations/instrumentation files. The reason it was silently failing was because `sed` commands don't have failure outputs unless the file it is targeting doesn't exist. As such, the release image was being tested which passes through the validation.

`'` is not missing, it's at the end of the `run:` value a few lines down. This string contains all the arguments needed to run the validator.

Reply:
See https://github.com/aws-observability/aws-otel-java-instrumentation/pull/652 for test runs that show the current logic will affect change in the outcome of the test. It is different because we implemented a complex retry mechanism on the enablement process for the canary.

Reply:
`testing:validator:` and `eks/` added due to change in how test are setup and run. Makes sense.